brands/shop/supermarket.json

```
@@ -1179,6 +1179,7 @@
   },
   "shop/supermarket|E. Leclerc Drive": {
     "countryCodes": ["fr"],
+    "matchNames": ["Leclerc Drive"]
```

thanks! don't forget the comma..
```suggestion
    "matchNames": ["leclerc drive"],
```